lms/djangoapps/teams/static/teams/js/teams_tab_factory.js

```diff
@@ -1,13 +1,23 @@
 ;(function (define) {
     'use strict';
-
-    define(['jquery', 'underscore', 'backbone', 'teams/js/views/teams_tab'],
-        function ($, _, Backbone, TeamsTabView) {
+    define([
+        'jquery',
+        'underscore',
+        'backbone',
+        'gettext',
+        'teams/js/views/teams_tab',
+        'teams/js/views/team_utils'
+    ], function ($, _, Backbone, gettext, TeamsTabView, TeamUtils) {
             return function (options) {
                 var teamsTab = new TeamsTabView({
                     el: $('.teams-content'),
                     context: options
                 });
+                $(document).ajaxError(function (event, xhr) {
+                    if (_.contains([401, 500], xhr.status)) {
+                        TeamUtils.showMessage(gettext("Your request could not be completed. Reload the page and try again."));
```

Comment:
@andy-armstrong We decided to go with a general, rather than a 401-specific, message. I'm not really sure how this could be factored out to be usable in other apps, since it's entirely reliant on the Teams-specific error message code.